scopes/workspace/workspace/pick-outdated-pkgs.ts

```diff
@@ -0,0 +1,130 @@
+import colorizeSemverDiff from '@pnpm/colorize-semver-diff';
+import semverDiff from '@pnpm/semver-diff';
+import { OutdatedPkg } from '@teambit/dependency-resolver';
+import { getBorderCharacters, table } from '@zkochan/table';
+import chalk from 'chalk';
+import { prompt } from 'enquirer';
```

Comment:
I don't think we have it as a dependency, so you probably need to add it. (it probably works because yarn hoist it to the root as a dependency of another package)
Or use inquirer (https://www.npmjs.com/package/inquirer) which we already have.

Reply:
added enquirer to the dependencies